docker/usr/local/bin/install-from-source

```diff
@@ -224,6 +224,7 @@ elif [ "${METHOD}" = 'install-meson-from-source' ]; then
         --cross-file="${PREFIX}/meson.cross" \
         --buildtype=release \
         --prefix="${PREFIX}" \
+        --pkg-config-path="${PKG_CONFIG_PATH}" \
```

Comment:
Is this a fix for the issue @ysims was having?

Reply:
Yep